src/test/shell/bazel/bazel_example_test.sh

```diff
@@ -151,15 +151,17 @@ function test_shell() {
 function test_python() {
   assert_build "//examples/py:bin"
 
-  ./bazel-bin/examples/py/bin >& $TEST_log \
+  env -u RUNFILES_DIR -u RUNFILES_MANIFEST_FILE \
```

Comment:
This was necessary, because the test itself also happens to drag in `//examples/py` from the Bazel source tree. This means that with the new logic added, the Python stub script would end up picking the wrong runfiles directory now.

I suspect that this part of the change is acceptable, because lookups of runfiles were already guaranteed to fail within the called process.

Reply:
Done!